tests/test_modeling.py

```
@@ -987,257 +909,236 @@ def test_loss_with_decoder_loss(self, mock_config):
+            "labels_attention_mask": torch.ones(C, 10, dtype=torch.long),
+            "words_mask": torch.randint(0, W, (B, L)),
+            "text_lengths": torch.tensor([W, W - 2]).unsqueeze(-1),
+            "span_idx": torch.randint(0, W, (B, L * max_width, 2)),
+            "span_mask": torch.randint(0, 2, (B, L * max_width)),
+            "labels": torch.zeros(B, L * max_width, C),
         }
-    
+
     def test_forward_output_shape_without_labels(self, mock_config, model_inputs):
         """Should return output with correct logits shape without labels."""
         from unittest.mock import Mock
```

Ruff (0.15.1)

[error] Redefinition of unused `Mock` from line 9: `Mock` redefined here

Remove definition: `Mock`

(F811)